prow/cmd/generic-autobumper/bumper/bumper.go

```diff
@@ -534,7 +534,20 @@ func upstreamConfigVersions(upstreamVersionType string, o *Options, parse func(u
 	return versions, nil
 }
 
-func parseUpstreamImageVersion(upstreamAddress string) (string, error) {
+func findExactMatch(body, prefix string) (string, error) {
+	for _, line := range strings.Split(strings.TrimSuffix(body, "\n"), "\n") {
+		if strings.Contains(line, prefix) {
+			res := imageMatcher.FindStringSubmatch(string(line))
```

Comment:
Instead of string splitting, I would prefer `imageMatcher.FindAllStringSubmatch` and filter through the matched results to find the first one containing `prefix`, this feels more accurate to me

Reply:
Discussed offline, but FindAllStringSubmatch would not work without major changes to the regex. We added another capture group to the regex to make the string splitting method more accurate